pydarn/utils/radar_pos.py

```diff
@@ -45,6 +45,8 @@
 from pydarn import SuperDARNRadars, gate2slant, Coords
```

Comment:
add under Modifications line the data, name, and you removed the virtual heigh models to another file. 
Example of this is in a below comment.